src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -587,25 +583,21 @@ def to_ingress(self, deployment: Deployment) -> Optional[V1AlphaIngressRoute]:
             spec=ingress_route_spec,
         )
 
-    def to_ingress_routes(
-        self, deployment: Deployment, https: bool
-    ) -> list[V1AlphaIngressRoute]:
+    def to_ingress_routes(self, deployment: Deployment) -> list[V1AlphaIngressRoute]:
         hosts = self.create_host_wrappers(deployment)
         return [
             V1AlphaIngressRoute.from_hosts(
                 metadata=self._to_object_meta(
-                    name=f"{host.name.lower()}-http{("s" if https else "")}-{i}",
+                    name=f"{host.name.lower()}-https-{i}",
```

Comment:
```suggestion
                    name=f"{host.name.lower()}-{i}",
```

We can take this opportunity to remove the `https` part from the ingress name and make this easier to read, making the current
```yaml
name: admin-dashboard-http-https-0
```
Become instead:
```yaml
name: admin-dashboard-http-0
```